hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -422,11 +425,23 @@ private Table newHmsTable(TableMetadata metadata) {
     Preconditions.checkNotNull(metadata, "'metadata' parameter can't be null");
     final long currentTimeMillis = System.currentTimeMillis();
 
+    String owner = metadata.properties().get(HiveCatalog.HMS_TABLE_OWNER);
+    if (owner == null) {
+      try {
```

Comment:
Since this is used in two places, can you make it a static method in a Hadoop util class?

Reply:
Ack